src/invidious/helpers/errors.cr

```diff
@@ -43,6 +43,8 @@ def error_template_helper(env : HTTP::Server::Context, status_code : Int32, exce
   # URLs for the error message below
   url_faq = "https://github.com/iv-org/documentation/blob/master/docs/faq.md"
   url_search_issues = "https://github.com/iv-org/invidious/issues"
+  url_search_issues += "?q=is:issue+is:open+"
+  url_search_issues += URI.encode_www_form("[Bug] " + issue_title)
```

Comment:
Minor nitpick: [Using string interpolation prevents an extra memory allocation](https://crystal-lang.org/reference/1.12/guides/performance.html#use-string-interpolation-instead-of-concatenation).
```suggestion
  url_search_issues += URI.encode_www_form("[Bug] #{issue_title}")
```

Reply:
that's a good nitpick to have! not entirely familiar with crystal yet so I appreciate the info.